internal/services/producer.go

```diff
@@ -108,7 +108,7 @@ func (ps *ProducerService) Refresh() {
 			"clientid": ps.canaryConfig.ClientID,
 		}
 		refreshMetadataError.With(labels).Inc()
-		glog.Errorf("Errors producer refreshing metadata: %v", err)
+		glog.Errorf("Error producer refreshing metadata: %v", err)
```

Comment:
Should this be more something like `Error refreshing metadata in producer` or something?

Reply:
Agree.